src/main/flight/mixer_tricopter.c

```diff
@@ -560,7 +560,6 @@ STATIC_UNIT_TESTED void tailTuneModeThrustTorque(thrustTorque_t *pTT, const bool
 			// uncomment for debugging
 			currentProfile->servoConf[6].max = pTT->servoAvgAngle.sum / pTT->servoAvgAngle.numOf;
 			currentProfile->servoConf[6].min = pTT->tailTuneGyroLimit * 10.0f;
-			saveConfigAndNotify();
```

Comment:
I had strange results with this one. Suspect it cannot do more than one save in the same loop.